lib/renderer-factory.js

```diff
@@ -55,13 +56,21 @@
      validateAtoms(atoms);
      cardOptions = cardOptions || {};
 
-     this.state = {
+     if (!dom) {
+       if (!window) {
```

Comment:
This check itself will throw `ReferenceError: window is not defined` in node.  `if (typeof window !== 'undefined')`